docs/api/non_geo_datasets.csv

```diff
@@ -0,0 +1,33 @@
+Dataset,Task,Source,# Samples,# Classes,Size (px),Resolution (m),Bands
+`ADVANCE`_,C,"Google Earth, Freesound","5,075",13,512x512,0.5,RGB
+`Benin Cashew Plantations`_,S,Airbus Pléiades,70,6,"1,186x1,122",0.5,MSI
+`BigEarthNet`_,C,Sentinel-1/2,"590,326",19--43,120x120,10,"SAR, MSI"
```

Comment:
Looking at the latest version of the docs preview, this is still showing up as 590326 with no comma. I'm not sure why, but I don't want it to hold up the PR. Will investigate at a future date.